### DISCUSSION TITLE
Tnx for great product!

### DISCUSSION THREAD

**Discussion by bob-tm** [custom-components/pyscript#188] [General]
I's take hours to write simple code using ha automations. 
Learn python from scratch and use pyscript is 100x faster then debug ugly ha automations. 

Tnx a lot!

**Comment by dorbitbrown** [comment]
Hear! Hear! Pyscript is awesome.

I was totally put off by the idea of writing automations, so I was VERY glad to find Pyscript.  Having now written a thousand lines of code, I can't imagine doing it any other way!
